iocore/eventsystem/I_EventProcessor.h

```diff
@@ -116,7 +128,7 @@ class EventProcessor : public Processor
     @return event object representing the start of the thread.
```

Comment:
I found not a single soul used that return.  However the docs then become misleading as they are.

This part will be separated out for other discussion.